src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -231,6 +231,15 @@ class PostTyper extends MacroTransform with IdentityDenotTransformer  { thisTran
                   ctx.compilationUnit.source.exists &&
                   sym != defn.SourceFileAnnot)
                 sym.addAnnotation(Annotation.makeSourceFile(ctx.compilationUnit.source.file.path))
+
+              for (parent <- sym.asClass.classInfo.classParents) {
```

Comment:
The comment on top of `class PostTyper` document eveything that the phase does, you should add something like `(10) Adds Child annotations to all sealed classes`